internal/validation/v2alpha1/rules.go

```diff
@@ -56,6 +58,24 @@ func validateRules(ctx context.Context, client client.Client, parentAttributePat
 	return problems
 }
 
+func validatePath(validationPath string, rulePath string) (problems []validation.Failure) {
+	problems = append(problems, validateEnvoyTemplate(validationPath+".path", rulePath)...)
+	return problems
+}
+
+func validateEnvoyTemplate(s string, path string) []validation.Failure {
+	segments := strings.Split(strings.TrimLeft(path, "/"), "/")
+	doubleAsteriskExists := false
+	for _, segment := range segments {
```

Comment:
If you iterate over the slice of segments, only last operator is important to you to check if it's glob match operator. If your path has only one count of `{**}` and If last operator is `{**}`, then it's a valid path. See https://go.dev/play/p/1N_Uk-Llm-j